alchy/utils.py

```diff
@@ -92,12 +92,12 @@ def get_mapper_class(model, field):
     return mapper_class(getattr(model, field))
 
 
-def merge_declarative_args(base_classes, config_key):
+def merge_declarative_args(cls, config_key):
     """Given a list of base classes, merge declarative args identified by
     `config_key` into a single configuration object.
     """
-    configs = [getattr(base, config_key, None)
-               for base in reversed(base_classes)]
+    configs = [base.__dict__.get(config_key)
+               for base in reversed(cls.__mro__)]
```

Comment:
I didn't know, but from the docs: ``This method can be overridden by a metaclass to customize the method resolution order for its instances. It is called at class instantiation, and its result is stored in __mro__.`` The way I read it I have a very slight preference for using ``__mro__``, but don't feel strongly at all. I can change it to use ``mro()`` if you prefer.